### DISCUSSION TITLE
Allow Passing Arbitrary Number of Lists to `zip`

### DISCUSSION THREAD

**Discussion by joealden** [remeda/remeda#965] [General]
Currently, `zip` only allows passing two lists (via it's two arguments). Lodash allows you to pass an arbitrary number of lists (see https://lodash.com/docs/4.17.15#zip).

Is there a reason why `remeda` doesn't support this, or is it implemented the way is it today due to the idea coming from Ramda initially (https://ramdajs.com/docs/#zip), or is it maybe due to the types likely getting more complex?

**Comment by cjquines** [comment]
see also #132
